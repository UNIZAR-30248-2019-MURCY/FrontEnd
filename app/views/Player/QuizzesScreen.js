import React, {Component} from 'react';
import {
    View,
    StyleSheet,
} from 'react-native'
import {Button, colors, Text} from 'react-native-elements';
import Icon from "react-native-vector-icons/FontAwesome";

export default class QuizzesScreen extends Component {

    constructor(props) {
        super(props);
    }

    render() {
        return (
            <View style={styles.container}>
                <View style={styles.containerTitle}>
                    <Text h2>Quizzes</Text>
                </View>

                <View style={styles.containerMessage}>
                    <Icon
                        name='wrench'
                        size={100}
                    />
                    <Text style={{color: 'grey', fontSize: 19, marginTop:20, textAlign:'center'}} >
                        We are working on it
                    </Text>
                </View>
            </View>

        )
    }
}

const styles = StyleSheet.create({
    container: {
        flex: 1,
        alignItems: 'center',
        //justifyContent: 'center',
        padding: 20
    },
    containerTitle: {

        marginTop: 50,
        padding: 20
    },
    containerMessage: {
        alignItems: 'center',
        //justifyContent: 'center',
        marginTop: 130,
        padding: 20
    },


})
