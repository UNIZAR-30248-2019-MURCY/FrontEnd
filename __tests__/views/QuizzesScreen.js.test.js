import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import QuizzesScreen from "../../app/views/Player/QuizzesScreen";

describe('<QuizzesScreen/>', () => {
    it('QuizzesScreen renders without crashing', () => {
        const rendered = renderer.create(<QuizzesScreen/>).toJSON();
        expect(rendered).toBeTruthy();
    });
    it('QuizzesScreen test against snapshot', () => {
        const welcomePage = renderer.create(<QuizzesScreen />).toJSON();
        expect(welcomePage).toMatchSnapshot();
    });
});
