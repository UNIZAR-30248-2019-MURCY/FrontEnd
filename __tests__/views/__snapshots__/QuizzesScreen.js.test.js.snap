// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<QuizzesScreen/> QuizzesScreen test against snapshot 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "padding": 20,
    }
  }
>
  <View
    style={
      Object {
        "marginTop": 50,
        "padding": 20,
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 42.5,
        }
      }
      theme={
        Object {
          "colors": Object {
            "disabled": "hsl(208, 8%, 90%)",
            "divider": "#bcbbc1",
            "error": "#ff190c",
            "grey0": "#393e42",
            "grey1": "#43484d",
            "grey2": "#5e6977",
            "grey3": "#86939e",
            "grey4": "#bdc6cf",
            "grey5": "#e1e8ee",
            "greyOutline": "#bbb",
            "platform": Object {
              "android": Object {
                "error": "#f44336",
                "primary": "#2196f3",
                "secondary": "#9C27B0",
                "success": "#4caf50",
                "warning": "#ffeb3b",
              },
              "ios": Object {
                "error": "#ff3b30",
                "primary": "#007aff",
                "secondary": "#5856d6",
                "success": "#4cd964",
                "warning": "#ffcc00",
              },
            },
            "primary": "#2089dc",
            "searchBg": "#303337",
            "secondary": "#8F0CE8",
            "success": "#52c41a",
            "warning": "#faad14",
          },
        }
      }
    >
      Quizzes
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "center",
        "marginTop": 130,
        "padding": 20,
      }
    }
  >
    <Text />
    <Text
      style={
        Object {
          "color": "grey",
          "fontSize": 19,
          "marginTop": 20,
          "textAlign": "center",
        }
      }
      theme={
        Object {
          "colors": Object {
            "disabled": "hsl(208, 8%, 90%)",
            "divider": "#bcbbc1",
            "error": "#ff190c",
            "grey0": "#393e42",
            "grey1": "#43484d",
            "grey2": "#5e6977",
            "grey3": "#86939e",
            "grey4": "#bdc6cf",
            "grey5": "#e1e8ee",
            "greyOutline": "#bbb",
            "platform": Object {
              "android": Object {
                "error": "#f44336",
                "primary": "#2196f3",
                "secondary": "#9C27B0",
                "success": "#4caf50",
                "warning": "#ffeb3b",
              },
              "ios": Object {
                "error": "#ff3b30",
                "primary": "#007aff",
                "secondary": "#5856d6",
                "success": "#4cd964",
                "warning": "#ffcc00",
              },
            },
            "primary": "#2089dc",
            "searchBg": "#303337",
            "secondary": "#8F0CE8",
            "success": "#52c41a",
            "warning": "#faad14",
          },
        }
      }
    >
      We are working on it
    </Text>
  </View>
</View>
`;
